pyproject.toml

```diff
@@ -16,20 +16,24 @@ version = "0.11.1"
 certifi = ">=2023.7.22"
 frozendict = "^2.3.2"
 pillow = "^9.0.0" # TODO: We may want to mark pillow (and numpy) as extra (https://python-poetry.org/docs/master/pyproject#extras)
-pydantic = "^1.7.4"
-python = ">=3.7.0,<4.0"
+pydantic = ">=2.0,<3.0.0"
+python = ">=3.9.0,<4.0"
 python-dateutil = "^2.8.2"
 requests = "^2.28.2"
 urllib3 = "^1.26.9"
 typer = "^0.9.0"
 
 [tool.poetry.group.dev.dependencies]
-datamodel-code-generator = "^0.12.1"
+datamodel-code-generator = "^0.22.1"
```

Comment:
Had to increase version in order to get this to play nice with other dependencies that require certain versions of pydantic.